src/renderer/views/Playlist/Playlist.js

```diff
@@ -291,6 +296,20 @@ export default defineComponent({
           this.getPlaylistInvidious()
           break
       }
+
+      if (
+        this.sortOrder === SORT_BY_VALUES.VideoLengthAscending ||
+        this.sortOrder === SORT_BY_VALUES.VideoLengthDescending
+      ) {
+        if (!this.shownNotification) {
+          for (const v in this.playlistItems) {
+            if (!Object.hasOwn(v, 'lengthSeconds') || isNaN(v.lengthSeconds)) {
+              showToast("This playlist contains at least one video that doesn't have a length, it will be sorted as if their length is zero.", 10000, () => {})
+              this.shownNotification = true
+            }
+          }
+        }
+      }
```

Comment:
Just call `this.checkPlaylist()` 
Also this part of code has no `break` = will shown N toasts for N videos without length (another reason not to just call existing method

Reply:
Yep, removing that code.